sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/eventsourcedentity/EventSourcedEntityHandler.scala

```diff
@@ -19,6 +19,12 @@ package com.akkaserverless.scalasdk.impl.eventsourcedentity
 import com.akkaserverless.scalasdk.eventsourcedentity.CommandContext
 import com.akkaserverless.scalasdk.eventsourcedentity.EventSourcedEntity
 
+object EventSourcedEntityHandler {
+  final case class CommandHandlerNotFound(commandName: String) extends RuntimeException
+
+  final case class EventHandlerNotFound(commandName: String) extends RuntimeException
```

Comment:
we should use the exceptions from the javasdk

`com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntityHandler.CommandHandlerNotFound` and same with EventHandlerNotFound

Reply:
Ah leftover, I'll remove this.